docs/dev/subscriptions.rst

```diff
@@ -30,3 +31,61 @@ If this is your first time setting up subscriptions, you will to re-sync djstrip
 The subscription settings (``RTD_PRODUCTS``) already mapped to match the Stripe prices from the test mode.
 To subscribe to any plan, you can use any `test card from Stripe <https://stripe.com/docs/testing>`__,
 for example: ``4242 4242 4242 4242`` (use any future date and any value for the other fields).
+
+Modeling
+--------
+
+Subscriptions are attached to an organization (customer),
+and can have multiple prices of products attached to it.
+A product can have multiple prices, usually monthly and yearly.
```

Comment:
So there's multiple prices of products, and also products can have multiple prices? I don't quite understand why we would have 2 different prices for the same product on a subscription?

Reply:
```suggestion
and can have multiple products attached to it.
A product can have multiple prices, usually monthly and yearly.
```

I think this is clearer?

Reply:
A subscription is attached to a price, not a product, but don't think you can have a subscription attached to several prices of the same product, yeah.